assets/src/dashboard/components/pageStructure/navigationComponents.js

```diff
@@ -19,50 +19,71 @@
  */
 import styled from 'styled-components';
 
+/**
+ * Internal dependencies
+ */
+import Button from '../button';
+
 export const Content = styled.div`
   display: flex;
   flex-direction: column;
-  padding: ${({ theme }) => theme.leftRail.contentPadding}px;
+  margin: 20px 0;
+  > * {
+    margin: 20px;
+  }
```

Comment:
this feels like too much space but when i look at the screen it seems _close_ I'm hoping that more eyes on this will correct anything I'm not seeing by just looking at the video/can start a convo.